tests/console/journalctl.pm

```diff
@@ -51,7 +56,7 @@ sub verify_journal {
     my $cmd = 'journalctl --verify';
     $cmd .= " --verify-key=$fss_key" if defined($fss_key);
 
-    script_run('journactl --flush');    # ensure data is written to persistent log
+    script_run('journalctl --flush');    # ensure data is written to persistent log
```

Comment:
I wonder what have been checking so far with such a fail....

Reply:
It should have ensured that we have all previous entries written to the file. So not a check of itself. More of prior preparation for following ones.